RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate method canOpenRoom to Typescript

## Proposed changes

Migrate method canOpenRoom to Typescript

## Issue(s)	


## How to test or reproduce

Try to open a a jitsi room via deeplink
Also try clicking on a link sent to the chat, to see if it opens jitsi or the specific chat.

## Screenshots

## Types of changes




## Checklist



## Further comments
